tracer/src/Datadog.Tracer.Native/Datadog.Tracer.Native.def

```diff
@@ -17,11 +17,12 @@ EXPORTS
     GetProbesStatuses
     DisableTracerCLRProfiler
     RegisterIastAspects
-    RegisterCallTargetDefinitions
+    RegisterCallTargetDefinitions3
```

Comment:
This seems to be exactly what we _don't_ want to do 🤔 We _shouldn't_ remove the `RegisterCallTargetDefinitions` method, it needs to remain there, unchanged. And we don't need `RegisterCallTargetDefinitions3` because it will never be called?

Reply:
Legacy init uses InitProfiler in version mismatch scenarios. RegisterCallTargetDefs was called when no mismatch was found, so not needed anymore (but for testing purposes)

Reply:
Ah ok, thanks 👍